### ISSUE TITLE
Gameplay

### ISSUE THREAD

**Issue by ghost** [haxball/haxball-issues#1180]
Hello, i have a windows 10 laptop and i had a windows 7 laptop before. But when i am playing with my w10 laptop gameplay does not feel good. İ have to increase exploration which causes lag. I mean player feels like sliding, game doesnt feel responsive. I was playing with 0 extra on my old laptop and game was pretty responsive. Like, 50 extra in this laptop equals 0 of old laptop. And this laptop has better specs,also.     I am playing really bad due to this lol :D
Note: After using extrapolation for a time, sometimes it feels like game resets extrapolation to 0. But actually it does not. Although it is 50 player feels like sliding.
Is there a way to fix this ?

**Comment by SMALIE** [comment]
[Haxapp](https://haxapp.site/) on this app you shouldnt have lags

**Comment by 9999OldSchool** [comment]
Of course, but the owner of this game thinks it's just because we all have input delay, but it's actually a fkn bug in haxball. One day it feels overpowered, another day as normal, with shitty little input delay. Then another day, completely shitty. It's just bug in the entire haxball that causes to use extrapolation, and still having input delay. I don't get it. Since extrapolation got added, it just started giving more and more input delays to more and more players. I can give you 100 proofs if needed. I won't even try to photoshop it because it's just real, haha. 

I tried to play on cheated 200 extrapolation and I had input delay, which is good for the owner of this game, but not for us players.
